src/com/lftechnology/java/training/alina/basic/NumberGuessingGame.java

```diff
@@ -0,0 +1,75 @@
+package com.lftechnology.java.training.alina.basic;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * Write a Guessing a number game application: application will generate a
+ * random no between 1 and 20. user guess the value by entering from the console
+ * Check if the guess value matched with random number or not, if matched show
+ * the guessing value, random value and the no. of attempts.
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ *
+ */
+public class NumberGuessingGame {
+
+	private static final Logger LOGGER = Logger.getLogger(NumberGuessingGame.class.getName());
+
+	public static void main(String[] args) {
+		int randomNumber;
+		Scanner scanner = null;
+		try {
```

Comment:
the other way to close the resource is using try-with-resource feature.
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
http://www.mkyong.com/java/try-with-resources-example-in-jdk-7/